src/python/pants/backend/python/macros/poetry_requirements_test.py

```diff
@@ -81,7 +82,44 @@ def test_max_tilde(test, exp) -> None:
     ],
 )
 def test_handle_str(test, exp) -> None:
-    assert parse_str_version("foo", test, "") == f"foo {exp}"
+    assert parse_str_version("foo", test, "", "") == f"foo {exp}"
+
+
+def test_add_markers() -> None:
+    # case with just mark
```

Comment:
Nit, these should be capitalized and with a period per Pants style guide: https://www.pantsbuild.org/v2.6/docs/style-guide.